### PR TITLE
Oner ince homework 3

### PR DESCRIPTION
**Pull request by OnerInce**

### CODE REVIEW

**Review by sahinof — APPROVED** [review]

File: ecommerce/baskets/serializers.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        fields = ("id", "basket", "product", "quantity", "price")
+
+
+class BasketDetailedSerializer(serializers.ModelSerializer):
+    customer = CustomerSerializer(many=False)
+
+    class Meta:
+        model = Basket
+        fields = ("customer", "status")
+
```
Comment (VeliAliyev) [inline ecommerce/baskets/serializers.py:27]: In detailed serializers, maybe it's better to inherit from  the non-detailed serializer. For example, here in BasketDetailedSerializer, if inherit from BasketSerializer you don't need to repeat class Meta.

File: ecommerce/baskets/serializers.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+class BasketItemSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = BasketItem
+        fields = ("id", "basket", "product", "quantity", "price")
+
+
+class BasketDetailedSerializer(serializers.ModelSerializer):
+    customer = CustomerSerializer(many=False)
+
```
Comment (VeliAliyev) [inline ecommerce/baskets/serializers.py:23]: In detailed serializers, maybe it's better to inherit from  the non-detailed serializer. For example, here in BasketDetailedSerializer, if inherit from BasketSerializer you don't need to repeat class Meta.

File: ecommerce/baskets/filters.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+from django.utils.translation import gettext_lazy as _
+from django_filters import rest_framework as filters
+
+from baskets.models import Basket, BasketItem
+from customers.filters import CustomerFilter
+from products.filters import ProductFilter
+
+
+class BasketFilter(filters.FilterSet):
+    customer = CustomerFilter
```
Comment (upy) [inline ecommerce/baskets/filters.py:10]: boyle bir tanimi nereden ogrendik? daha once boyle bir tanim gormemistim.

File: ecommerce/baskets/serializers.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    class Meta:
+        model = Basket
+        fields = ("customer", "status")
+
+    @atomic()
+    def create(self, validated_data):
```
Comment (upy) [inline ecommerce/baskets/serializers.py:29]: detailed serializer'lari genelde create/update icin (ozel durumlar haric) kullanmamaya calisiyoruz, ayrica basket olusturulurken customer olusturmak cok kullanisli degil sanki?

File: ecommerce/orders/serializers.py [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+
+class OrderItemSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrderItem
+        fields = ("id", "order", "product", "price",)
+
+
+class BillingAddressDetailedSerializer(serializers.ModelSerializer):
+    city = CitySerializer(many=False)
```
Comment (upy) [inline ecommerce/orders/serializers.py:43]: default many degeri false olmasi lazim, bu nedenle tanim yapmamiza gerek yok sanki?

File: ecommerce/orders/serializers.py [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+class OrderBankAccountDetailedSerializer(serializers.ModelSerializer):
+    order = OrderSerializer(many=False)
+
+    class Meta:
+        model = OrderBankAccount
+        fields = ("id", "name", "iban", "bank_name", "order",)
+
+    @atomic()
+    def create(self, validated_data):
```
Comment (upy) [inline ecommerce/orders/serializers.py:142]: endpointi kullanarak hic buradaki methodu call edebildiniz mi?

**Review by upy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**